drivers/adc/ad7768-1/ad77681.c

```diff
@@ -46,6 +46,7 @@
 #include <string.h>
```

Comment:
Could you add a short description to the commit about why this change is needed?

Reply:
Done adding description.